neqo-http3/src/recv_message.rs

```diff
@@ -304,6 +321,23 @@ impl RecvMessage {
             RecvMessageState::ClosePending | RecvMessageState::Closed
         )
     }
+
+    fn is_interim(&self, headers: &[Header]) -> Res<bool> {
+        match self.message_type {
+            MessageType::Response => {
+                let status = headers.iter().find(|e| e.0 == ":status");
+                if let Some(s) = status {
```

Comment:
Don't use numeric tuple fields if you can avoid it.

```suggestion
                if let Some((_name, value)) = status {
```